docs/variants/dell_optiplex/recovery.md

```diff
@@ -234,3 +234,153 @@ Use following procedure if your 4M flash contain vendor BIOS.
 ```bash
 flashrom -w your_bios_backup_8M.bin -p linux_spi:dev=/dev/spidev1.0,spispeed=16000 -c "MX25L6406E/MX25L6408E"
 ```
+
+# RaspberryPi Recovery
+
+## Overview
+
+If you attempted to flash dasharo, using the below command, without using
+the Dasharo Tools Suite OEM you may have caused a break within the Intel ME
+region of the flash chip.
+
+```bash
+flashrom -p internal --ifd -i bios -i me -w <dasharo_optiplex_9010_firmware>
+```
+
+## Required Items
+
+You will need to purchase an
+[8 Pin Pomona Clip](https://www.digikey.com/en/products/detail/pomona-electronics/5250/745102?s=N4IgTCBcDaIKxjgBgAQAUD2BbDA7AhiiALoC%2BQA),
+a [16 Pin Pomona Clip](https://www.digikey.com/en/products/detail/pomona-electronics/5252/745103?s=N4IgTCBcDaIKxgQAgAoHsC2aB2BDJIAugL5A),
+a [Raspberry pi 3B+ or above](https://www.adafruit.com/product/3775?src=raspberrypi),
+and [female-to-female DuPont cables](https://www.amazon.com/40pcs-Female-2-54mm-Jumper-2x40pcs/dp/B00GSE2S98)
+
+## Steps
+
+1. Remove heatsink from CPU. [Linked video with correct timestamp](https://youtu.be/oFu3zpvo1Vg?t=240)
+
+1. Plug your DuPont female-to-female connectors to the RPi as displayed below.
+
+  Edge of pi (furthest from you)
+              (UART)
+             TX
+L         GND | RX                    CS
+E          |  |  |                    |
+F +--------------------------------------------------------------+
+T |  x  x  x  x  x  x  x  x  x  x  x  x  x  x  x  x  x  x  x  x  |
+  |  x  x  x  x  x  x  x  x  x  x  x  x  x  x  x  x  x  x  x  x  |
+E +--------------------------^--^--^--^-----------------------^--+
+                             |  |  |  |                       20
+D                     (VCC)-3.3V| MISO|                       |
+G                             MOSI    CLK                    GND
+E
+  Body of Pi (closest to you)
```

Comment:
I would put it into fenced code block to keep the spacing and character width consistent